doc/snek.adoc

```diff
@@ -850,13 +879,14 @@ separating out names and keywords from operators and other syntax.
 
 Snek uses indentation to identify program structure. Snek does not
 permit tabs to be used for indentation; tabs are invalid characters in
-Snek programs. Statements in the same block are indented the same
+Snek programs. Statements in the same block (list of
+statements) are indented the same
 amount; statements in deeper blocks are indented more, statements in
-external blocks less.
+shallower blocks less.
 
-When typing Snek directly at the Snek prompt, blank lines become
-significant as Snek cannot know what you will type next. You can see
-this in the Tutorial where Snek finishes an indented block at the
+When typing Snek directly at the Snek prompt blank lines become
+significant, as Snek cannot know what you will type next. You can see
+this in the Tutorial, where Snek finishes an indented block at the
```

Comment:
Yep. I think it's fine to mention it again here.

Reply:
well, the trouble is that it's really unclear above, having confused you into thinking it should have said '> ' instead of '+ '. I think this text is fine, but the earlier text needs works.